src/Emogrifier/CssInliner.php

```diff
@@ -399,7 +364,6 @@ protected function process()
 
         // grab any existing style blocks from the html and append them to the existing CSS
         // (these blocks should be appended so as to have precedence over conflicting styles in the existing CSS)
-        $allCss = $this->css;
         if ($this->isStyleBlocksParsingEnabled) {
             $allCss .= $this->getCssFromAllStyleNodes($xPath);
         }
```

Comment:
This modifies the parameter passed to the function.  Should the parameter be a separate entity (e.g. `$css`) with the preceding line being changed to `$allCss = $css;` rather than removed?

Reply:
Done.